remove unnecessary setting for original dst

ramaraochavali opened the pull request:
For original DST cluster, we already handle it in applyTrafficPolicy. This is not needed now

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ X] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure


Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[X ] Does not have any changes that may affect Istio users.

howardjohn reviewed (commented):
I think the early return is important. Otherwise we may hit settings below, such as the redis one

ramaraochavali commented:
Yeah, I realized that early return is needed after pushing the PR. I thought I updated it but missed to do that. Thanks for catching it. Fixed it now. PTAL
